pkg/globalnet/controllers/ipam/ipam.go

```diff
@@ -364,19 +364,19 @@ func (i *Controller) handleUpdateNode(old, newObj interface{}) {
 		return
 	}
 
-	oldCniIfaceIpOnNode := old.(*k8sv1.Node).GetAnnotations()[constants.CniInterfaceIP]
-	newCniIfaceIpOnNode := newObj.(*k8sv1.Node).GetAnnotations()[constants.CniInterfaceIP]
-	if oldCniIfaceIpOnNode == "" && newCniIfaceIpOnNode != "" {
-		klog.V(log.DEBUG).Infof("In handleUpdateNode, node %q is now annotated with cniIfaceIP, enqueing", newObj.(*k8sv1.Node).Name)
+	oldCniIfaceIPOnNode := old.(*k8sv1.Node).GetAnnotations()[constants.CniInterfaceIP]
```

Comment:
I guess the linter doesn't flag "Cni"? This is a bit of a strange var name, it expands to include "interface interface". I guess this is out of scope for this PR.

Reply:
I have only changed the names that the linter flagged.

Reply:
CNI isn't a well-known acronym outside k8s, like ID and IP.

Reply:
We can add our own configuration with other acronyms we want to watch for...